ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -87,15 +87,12 @@ export const commonStyles = reactStyles({
     borderRadius: '18px',
     transform: 'rotate(270deg)',
   },
-  incompleteModule: {
-    color: colorWithWhiteness(colors.highlight, -0.2),
+  incompleteOrExpiringModule: {
+    color: colors.warning,
```

Comment:
should this be bold as well? it appears like that in https://projects.invisionapp.com/share/C5121MQVD2ZB#/screens/464159071

Reply:
I'm not sure.  It seems odd that only that one status would be bold.  Good discussion Q!

Reply:
Trying out a bold treatment in #6333